its_live_monitoring/src/sentinel2.py

```diff
@@ -85,19 +91,32 @@ def get_sentinel2_stac_item(scene: str) -> pystac.Item:
 def qualifies_for_sentinel2_processing(
     item: pystac.Item,
     *,
+    reference: pystac.Item = None,
     max_cloud_cover: int = SENTINEL2_MAX_CLOUD_COVER_PERCENT,
     log_level: int = logging.DEBUG,
 ) -> bool:
     """Determines whether a scene is a valid Sentinel-2 product for processing.
 
     Args:
         item: STAC item of the desired Sentinel-2 scene.
+        reference: STAC item of the Sentinel-2 reference scene.
```

Comment:
Might be worth noting that this is optional and only for relative orbit comparison.
```suggestion
        reference: STAC item of the Sentinel-2 reference scene for optional relative orbit comparison.
```